test/testsuites/packagestests.go

```diff
@@ -84,6 +91,9 @@ func childResourcesNotCreated(t *testing.T) {
 // testOpSrcWithConflictingPackages ensures that an OperatorSource and its child resources
 // are successfully rolled out regardless of whether or not there is a
 // package conflict with an exiting OperatorSource.
+//
+// This test also checks that a CSC with an unspecified source cannot be resolved if its
+// packages exist in two OperatorSources.
```

Comment:
This test has become ambiguous to me now. Maybe you should split it into two?

```
// testOpSrcWithConflictingPackages ensures that an OperatorSource and its child resources
// are successfully rolled out regardless of whether or not there is a
// package conflict with an exiting OperatorSource.
```

Is this true? Shouldn't this result in a error?

Reply:
This description matches marketplace behavior. If we want to change that behavior we could create a JIRA in the backlog.

Reply:
I can certainly split it into two

Reply:
>This description matches marketplace behavior. If we want to change that behavior we could create a JIRA in the backlog.

OK, that description is hard to parse. Is it saying that I can add two OperatorSources that have some identical packages?

Reply:
That's correct. I can update the comment to better reflect that message.